src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -30,9 +30,30 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   </Choose>
 
   <PropertyGroup>
-    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <EnableDefaultCompileItems Condition=" '$(EnableDefaultCompileItems)' == '' ">false</EnableDefaultCompileItems>                                <!--- Do not glob F# source files -->
+    <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">false</EnableDefaultNoneItems>
+  </PropertyGroup>
+
+  <PropertyGroup>
     <DefaultProjectTypeGuid Condition=" '$(DefaultProjectTypeGuid)' == '' ">{F2A71F9B-5D33-465A-A702-920D77279786}</DefaultProjectTypeGuid>         <!-- F# project type -->
+    <DocumentationFile>bin\$(Configuration)\$(AssemblyName).XML</DocumentationFile>
```

Comment:
If someone changes the outputpath to some other folder then this will be out of sync with the output assembly. Given the SDK defaults this you can remove this as well. I would also remove DebugType as well.